polygeist/tools/cgeist/Test/Verification/assume.cpp

```diff
@@ -0,0 +1,42 @@
+// RUN: cgeist -O0 %s -w -o - -S --function=* | FileCheck %s
+
+// The function being called here should not be generated as it won't be called.
+// CHECK-NOT: func.func{{.*}}@{{.*}}g
+
+bool g();
+
+// Assume with a boolean variable as an argument
+
+// CHECK-LABEL:   func.func @_Z2f0b(
+// CHECK-SAME:                      %[[VAL_0:.*]]: i1)
+// CHECK:           "llvm.intr.assume"(%[[VAL_0]]) : (i1) -> ()
```

Comment:
Should we be using `CHECK-NEXT` here to make sure no other, unexpected instructions are generated?